frontend/packages/console-app/src/components/console-operator/ConsoleOperatorConfig.tsx

```diff
@@ -80,6 +83,43 @@ const ConsolePluginStatus: React.FC<ConsolePluginStatusType> = ({ enabled, plugi
 
 const ConsolePluginsList: React.FC<ConsolePluginsListType> = ({ obj }) => {
   const { t } = useTranslation();
+  const toastContext = useToast();
+  const [pluginsData, pluginsError] = useURLPoll<ConsolePluginsListResponseType>(
+    `${window.SERVER_FLAGS.basePath}api/plugins`,
+  );
+
+  const prevPluginsDataRef = React.useRef<ConsolePluginsListResponseType>();
+  React.useEffect(() => {
+    prevPluginsDataRef.current = pluginsData;
+  });
+  const prevPluginsData = prevPluginsDataRef.current;
+
+  const pluginsStateInitialized = _.isEmpty(pluginsError) && !_.isEmpty(prevPluginsData);
+  const pluginsChanged = !_.isEqual(prevPluginsData?.plugins, pluginsData?.plugins);
```

Comment:
I'd suggest `_.xor` so that the order of the plugins doesn't matter. Maybe

```suggestion
  const pluginsChanged = !_.isEmpty(_.xor(prevPluginsData?.plugins, pluginsData?.plugins));
```